styles/styles.css

```diff
@@ -0,0 +1,155 @@
+header {
+  background-image: url('../assets/hero.jpg');
+  /*background-position: 50% 50%;*/
+  background-size: cover;
+  background-repeat: no-repeat;
+  border-bottom: solid;
+  border-color: #9D9494;
+  height: 100px;
+  color: #93737F;
```

Comment:
This is a general protip (that you may already know) - do you have color-picker installed on Atom? If you press CMD-SHIFT-C it'll open up the little color picker window and it's super amazing 🌈 ✨